src/modules/Topics/Topics.tsx

```diff
@@ -108,19 +104,21 @@ const Topics: React.FC = () => {
     <Card className='kafka-ui-m-full-height' data-ouia-page-id='tableTopics'>
       <KafkaTopics
         topics={topicsList}
-        getUrlFortopic={(row) => `${basename}/topics/${row.topic_name}`}
-        onDelete={(row) => onDelete(row.topic_name)}
-        onEdit={(row) => onEdit(row.topic_name)}
-        onSearchTopic={(value) => {
+        getUrlFortopic={(row: { name: any }) =>
```

Comment:
this typing shouldn't be required, it should be inferred by Typescript automatically